test/__snapshots__/app.spec.js.snap

```diff
@@ -61,6 +61,6 @@ exports[`App component only should be mounted correctly 1`] = `
       />
     </Switch>
   </Main>
-  <IconSprite />
+  <UNDEFINED />
```

Comment:
According to :
https://github.com/cozy/cozy-ui/releases/tag/v52.0.0
 there is a Breaking change here

Reply:
So you've to change the way you import the Sprite. Snapshot can be useful in fact :D

Reply:
@Crash-- indeed. I updated.
I think cozy-settings wasn't updated in quite some time, it was far harder to update than the other cozy-apps (some of them did not have a single issue).